src/main/java/com/jlr/ttl/ds/api/util/mapper/LocationTypeValueMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.jlr.ttl.ds.api.util.mapper;
```

Comment:
Delete this file

Reply:
All the files are out of this ticket's scope have been removed.